src/common/components/page-links/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component, PropTypes } from 'react';
+
+import styles from './styles.css';
+
+const LINK_LABEL_OPTIONS = ['first', 'prev', 'next', 'last'];
+
+export default class PageLinks extends Component {
+
+  render() {
+    const renderLink = this.renderLink.bind(this);
+
+    let links = LINK_LABEL_OPTIONS.map((item) => {
+      return renderLink(
+        this.props[item],
+        item
+      );
+    });
+
+    return (
+      <ul className={ styles.container }>
+        { links }
+      </ul>
+    );
+  }
+
+  renderLink(item, label) {
+    let url;
+
+    if (typeof item == 'string') {
```

Comment:
This sort of comment should probably be a comment in the source code itself rather than being buried in a PR and hard to find in future.  (Ditto other similar comments.)

What happens if the item is something other than a string or undefined?  That looks like it'll render an `<a>` element with `href` set to undefined; I don't know off the top of my head what that will do.